src/subscription_manager/managercli.py

```diff
@@ -2965,6 +2994,16 @@ def _do_command(self):
         return result
 
 
+class TokenCommand(CliCommand):
+    """
+        Token Option with register command
+    """
+
+    def __init__(self):
+        shortdesc = _("Generate an offline token for authentication")
+        super(TokenCommand, self).__init__("token", shortdesc, False)
```

Comment:
Is this command work in progress? It doesn't do anything.

Reply:
@jirihnidek , I will work on your suggestions